studioShared/schemas/documents/blogPosts.ts

```diff
@@ -0,0 +1,69 @@
+import { defineField, defineType } from "sanity";
+import { format, parseISO } from "date-fns";
+import { richText, title } from "studio/schemas/fields/text";
+import { titleSlug } from "studio/schemas/schemaTypes/slug";
+
+export const blogPostsID = "blogPosts";
+
+const blogPosts = defineType({
+  name: blogPostsID,
+  type: "document",
+  title: "Blog posts",
+  fields: [
+    title,
+    titleSlug,
+    defineField({
```

Comment:
set this as shared schema

Reply:
looked into it, I don't think we need this just yet as its only used in post here in shared